build-utils/protocol-base-mspec/src/main/antlr4/org/apache/plc4x/plugins/codegenerator/language/mspec/MSpec.g4

```diff
@@ -152,6 +152,10 @@ dataType
  | base='dateTime'
  ;
 
+ENDIANNESS_LITERAL
+ : 'big endian' | 'little endian'
+ ;
+
```

Comment:
Could we make this "LE" or "BE"?

Reply:
Not sure if you want LE/BE instead of little endian and big endian, or next to i? I took longer notation as it is self descriptive. Making BE/LE option next to these isn't a problem as I extracted comparison to one place which works with antlr token regardless of its use context (type, field etc).